CenterForOpenScience/osf.io
# Feature/admin userview preprint [ENG-414]

## Purpose

Adding the ability to view Preprints on the admin app user view. Also adding pagination to preprints and nodes on the admin app user view. 

## Changes
### All Changes were conducted in the admin/users folder

- Updated views.py UserView class to display preprints and modified how nodes were being displayed.
- Updated user.html. Modified the template for the preprint display and pagination for preprint and nodes.
- Updated serializers.py to display preprints and render nodes differently for pagination.
- Created node_preprint_paginated_list.html as a template for preprints and nodes and the user view page.
## QA Notes

This modified the admin app user view. 
Verify pagination behavior of user nodes and preprints.
"Restore" button for deleted nodes has been removed from this template from product.

## Documentation


## Side Effects



## Ticket

https://openscience.atlassian.net/browse/ENG-414
